source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -113,6 +113,16 @@ destination cluster.
 Chunk Distribution
 ''''''''''''''''''
 
+.. important::
+   
+   Even if the source cluster is balanced, ``mongosync`` does not
```

Comment:
[nit] We can use contractions in our docs for simplicity!
```suggestion
   Even if the source cluster is balanced, ``mongosync`` doesn't
```

Same goes for the instance below!